### DISCUSSION TITLE
Homepage displays "Nothing is trending right now"

### DISCUSSION THREAD

**Discussion by NicolaeNMV** [mastodon/mastodon#21206] [Q&A]
Hello,

I've installed my own instance yesterday and was wondering why the homepage doesn't display anything.

It just says "Nothing is trending right now". I do have my own user that seems to work.

My instance is at https://mastodonmd.com.

<img width="937" alt="Screenshot 2022-11-20 at 11 08 48" src="https://user-images.githubusercontent.com/224201/202896393-1cf93f5d-dd7e-4a58-9d7b-731f771744d2.png">

**Comment by mjankowski** [comment]
Based on this being three years ago and the server no longer being online, I'm going to guess this is not relevant anymore.

That said, when you first set up a server there will not be activity here because the DB is empty (nothing trending, no content at all really).

As you follow accounts, tags, other servers, etc - assuming the tasks to update trends are running - you will start seeing stuff there (and the admins will be notified about that).
